src/features/auth/authSlice.js

```diff
@@ -43,6 +44,14 @@ export const login = createAsyncThunk(
 
 export const logout = createAction("auth/logout");
 
+export const getAllUsers = createAsyncThunk("auth/getAllUsers", async () => {
+  const {
+    data: { users },
+  } = await axios.get("/api/users");
+  console.log(users);
```

Comment:
You can remove it if you don't need it anymore.

Reply:
done